source/parameters/aperture.md

```diff
@@ -2,28 +2,30 @@
 ## ApertureP: Aperture Parameters
 
 The `ApertureP` parameter group contains parameters for describing an aperture. 
-The components of this group are:
+The components of this group and their defaults are:
 ```{code} yaml
-x_limits                     # [m] Vector of two real numbers
-y_limits                     # [m] Vector of two real numbers
-shape                        # Switch
-location                     # Switch
-aperture_shifts_with_body    # Boolian
-vertices                     # Structure
-material                     # String
-thickness                    # [m] Real number
+refp1:                            # [string] user-defined name
+  kind: ApertureP
+  x_limits: 0                     # [m] Vector of two real numbers
+  y_limits: 0                     # [m] Vector of two real numbers
+  shape: ""                       # [string] Aperture shape switch
+  location: ENTRANCE_END          # [string] Aperture location switch
+  aperture_shifts_with_body: ...  # [Boolean] ... TODO: describe ... TODO: default
```

Comment:
There is a name mismatch.  The`misalignment_moves_aperture` discussed below this should be renamed to `aperture_shifts_with_body`.

Reply:
Correct.